driver-mapping/pom.xml

```diff
@@ -108,11 +109,37 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.codehaus.groovy</groupId>
+            <artifactId>groovy-all</artifactId>
+            <version>${groovy.version}</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Keeping [JAVA-1296](https://datastax-oss.atlassian.net/browse/JAVA-1296) in mind as well, i'm also wondering about that.   Testing all sorts of different language runtimes could get kind of overbearing after a while.  wdyt about a separate module for these kinds of one off tests?